demo/res/raw/featurecollection_geojson.json

```diff
@@ -0,0 +1,36 @@
+{ "type": "FeatureCollection",
+"bbox": [-180.0, -90.0, 10.0, 90.0],
+    "features": [
+      { "type": "Feature",
+        "geometry": {"type": "Point", "coordinates": [102.0, 0.5]},
+        "properties": {"id": "ID", "draggable": true, "title": "abcdedf", rotation: 50.5}
+        },
+      { "type": "Feature",
+        "geometry": {
+          "type": "LineString",
+          "coordinates": [
+            [102.0, 0.0], [103.0, 1.0], [104.0, 0.0], [105.0, 1.0]
+            ]
+          },
+        "properties": {
+          "prop0": "value0",
+          "prop1": 0.0
+          }
+        },
+      { "type": "Feature",
+         "geometry": {
+           "type": "Polygon",
+           "coordinates": [
+                [ [100.0, 0.0], [101.0, 0.0], [101.0, 1.0], [100.0, 1.0], [100.0, 0.0] ],
+                [ [100.2, 0.2], [100.8, 0.2], [100.8, 0.8], [100.2, 0.8], [100.2, 0.2] ]
+            ]
+         },
+         "properties": {
+           "prop0": "value0",
+           "prop1": {},
+           "geodesic": true,
+           "fill color": 0xff00ff
```

Comment:
A couple of things in this file - hex notation is not supported in JSON, so either stringify or use the integer format.

Also, above you have "rotation: 50.5" and 'rotation' is not quoted. You'll need to quote it too.

Bonus points if this file is parsed correctly anyway :)